browser/src/UI/State.ts

```diff
@@ -72,6 +72,16 @@ export interface IState {
     activeWindowDimensions: Rectangle
 
     activeMessageDialog: IMessageDialog
+    commandLine: ICommandLine | null
+}
+
+export interface ICommandLine {
+    content: Array<[any, string]>,
```

Comment:
Cool, thanks for thinking about how to type this. Some of the cases from Neovim are tricky!